crates/uv-configuration/src/project_build_backend.rs

```diff
@@ -24,6 +24,10 @@ pub enum ProjectBuildBackend {
     #[serde(alias = "pdm-backend")]
     #[cfg_attr(feature = "clap", value(alias = "pdm-backend"))]
     PDM,
+    /// Use [poetry-core](https://pypi.org/project/poetry-core) as the project build backend.
+    #[serde(alias = "poetry-core")]
+    #[cfg_attr(feature = "clap", value(alias = "poetry-core", alias = "poetry_core"))]
+    PoetryCore,
```

Comment:
I'd call this entry `Poetry` and keep the poetry core variants alias, similar to hatch/hatchling

Reply:
So, something like this?
```rs
    #[serde(alias = "poetry-core")]
    #[cfg_attr(feature = "clap", value(alias = "poetry-core", alias = "poetry_core"))]
    Poetry,
```

Reply:
yep!